src/main/java/ee/tuleva/onboarding/fund/response/FundResponse.java

```diff
@@ -23,10 +23,10 @@
     private Fund.FundStatus status;
 
 
-    public FundResponse(Fund fund, PensionFundStatistics pensionFundStatistics) {
+    public FundResponse(Fund fund, PensionFundStatistics pensionFundStatistics, String language) {
         this.fundManager = fund.getFundManager();
         this.isin = fund.getIsin();
-        this.name = fund.getName();
+        this.name = "et".equalsIgnoreCase(language) ? fund.getNameEstonian() : fund.getNameEnglish();
```

Comment:
You could create a method in the `Fund` class like `fund.getName(language)` to avoid duplicating this if-else check.